source/documentation/requirements.html.md

```diff
@@ -201,9 +207,18 @@ be requested opening a [bug in bugzilla for RHEL product](https://bugzilla.redha
 This bug will be evaluated following the RHEL process.
 
 * For packages installed from RDO CloudSIG repos, the package must be updated in Fedora first to the required
-version. If it has not been updated first you can contact Fedora package maintainer or open a [bug for Fedora
-product](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora). Once the package has been updated, yo can
-send a [request to get it updated in RDO repos](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution).
+version. If it has not been updated first you can contact Fedora package maintainer or search for open bug against
+component, example [component:python-migrate](https://bugzilla.redhat.com/buglist.cgi?product=Fedora&component=python-migrate&bug_status=__open__), if bug not exist,open a [bug for Fedora product](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora).
+Once the package has been updated, you can send a [request to get it updated in RDO
+repos](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution).
+
```

Comment:
I think we could drop opening this BZ ?

Reply:
Yes, it makes sense to drop the BZ if the reporter is opening the review him/herself.

Reply:
Done